### PR TITLE
LPS-165768 | master

### PR DESCRIPTION
**Pull request by weller0246**
[LPS-165768](https://issues.liferay.com/browse/LPS-165768)

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase [hunk 1]
```diff
@@ -376,6 +370,37 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-150388 TC-1/5: Admin password need to be set on first startup if the default admin password property is not set."
+	@ignore = "true"
```
Comment (ling-alan-huang) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase:374]: Trailing tab?

**Comment by weller0246** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by weller0246** [comment]
ci:forward

**Comment by weller0246** [comment]
ci:stop

**Comment by weller0246** [comment]
ci:forward

**Pull request closed without merging**